src/sagemaker/serve/utils/telemetry_logger.py

```diff
@@ -79,7 +84,11 @@ def wrapper(self, *args, **kwargs):
                         None,
                         extra,
                     )
-            except ModelBuilderException as e:
+            except (
```

Comment:
We should classify endpoints that don't flip to `In Service` as failed from a telemetry perspective

Reply:
Got it.